File: ProcessVisualization.Api.Contracts/DataTransferObjects/Room/RoomDetailsViewDto.cs
using ProcessVisualization.Api.Contracts.DataTransferObjects.Documents;
using ProcessVisualization.Api.Contracts.DataTransferObjects.User;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProcessVisualization.Api.Contracts.DataTransferObjects.Room
{
    public class RoomDetailsViewDto : RoomViewDto
    {
        public List<UserDto> Users { get; set; } 
        public List<DocumentDto> Documents { get; set; }
    }
}

File: ProcessVisualization.Client/src/dtos/rooms/room-details-view.dto.ts
import { DocumentDto } from "src/dtos/documents/document.dto";
import { UserDto } from "src/dtos/user/user.dto";
import { RoomViewDto } from "./room-view.dto";

export class RoomDetailsViewDto extends RoomViewDto {
  Users: UserDto[] = [];
  Documents: DocumentDto[] = [];

  constructor() {
    super();
  }
}
